coral/src/app/components/Form.tsx

```diff
@@ -145,6 +145,44 @@ export const TextInput = <T extends FieldValues>(
 
 TextInput.Skeleton = BaseInput.Skeleton;
 
+//
+// <NumberInput>
+//
+function _NumberInput<T extends FieldValues>({
+  name,
+  formContext: form,
```

Comment:
Question: Why do we sometimes use 
```ts
 const ctx = useFormContext<T>();
```
to get the `formContext`, and sometimes pass it as a prop?

Reply:
The "base" comes from the upstream project and I simply replicatd that logic to the new field.

Reply:
Now that I look at the code: AFAIK we always get the form context and pass it down to the helper components. At least we are doing to for: `PasswordInput`, `TextInput`, `NumberInput`, `SubmitButton`